Marlin/pins_ZRIB_V20.h

```diff
@@ -0,0 +1,20 @@
+  #include "pins_MKS_13.h"
+#include "pins_RAMPS.h"
+  #undef  SERVO1_PIN 6

```

Comment:
`#undef` doesn't take a value. If you mean to redefine it, you should use both `#undef` and `#define`.